tst/launch_ci_runner.py

```diff
@@ -177,15 +184,28 @@ def run_tests_in_temp_dir(pr_number, head_repo, head_ref, commit_sha):
                     universal_newlines=True,
                 )
 
+            # Build output path and create directory if necessary
+            username = os.getenv("USER")
+            output_dir = os.path.join(
+                "/usr",
+                "projects",
+                "jovian",
+                "ci",
+                username,
+                f"pr_{args.pr_number}",
```

Comment:
There are a couple competing issues here:
- Multiple users running the CI script on the same could overwrite each others' artifacts (potentially with different commits)
- We don't want to leave every CI run's artifacts in `/usr/projects/jovian` forever
- We want anyone to be able to quickly find the latest CI output regardless of user (an issue with my current approach)

What about writing outputs to the directory `/usr/projects/jovian/ci/pr_[pr number]/[date]_[time]` and pairing that with a `/usr/projects/jovian/ci/cleanup.py` script anyone can call that will erase all `[date]_[time]` folders older than seven days? The latter script could be linked to a cron job except I'm not sure that we users can create cron jobs on darwin so it might have to be run manually every so often.

Reply:
yes, `/usr/projects/jovian/ci/pr_[pr number]/[date]_[time]` is what I was thinking of.

Reply:
Fixed:
```
[brryan@darwin-fe1 2024-12-02_12:54:58]$ pwd
/usr/projects/jovian/ci/pr_22/2024-12-02_12:54:58
```